docs/crio.8.md

```diff
@@ -97,6 +102,9 @@ set the CPU profile file path
 **--root**=""
   CRIO root dir (default: "/var/lib/containers/storage")
 
+**--registry**=""
+  List of registries to be used when pulling an unqualified image
```

Comment:
This is the same as insecure registries flag, so it's a flag which can be specified multiple times.